word_guess.rb

```diff
@@ -0,0 +1,182 @@
+#IN-CLASS - THURS FEB 25
+#game similar to hangman - cat loses z if wrong guess
+#game over shows cat leaving
+
+class WordGuess
+  attr_accessor :guessed_letters, :letter, :answer, :underscores, :num, :good,
+  :cat_message, :sleepy_cat, :bike_cat, :words
+
+  def initialize
+    @words = ["loon", "unicorn", "baboon", "ocelot", "dachshund", "penguin",
+      "cuttlefish","greyhound", "meerkat", "rhinoceros", "giraffe", "slug",
+      "zonkey", "narwhal", "opossum", "emu", "wombat", "duck"].shuffle
+    @guessed_letters = [""]
+    @letter = letter
+    @answer = @words[0]
+    @underscores = underscores
+    @num = 0
+    @cat_message = cat_message
+    @sleepy_cat = sleepy_cat
+    @bike_cat = bike_cat
+  end
+
+  def if_statement
```

Comment:
I think it would be better to name this method with the purpose as opposed to the type of logic contained